cmdexec/js.wasm.go

```diff
@@ -0,0 +1,23 @@
+//go:build wasm
+
+package cmdexec
+
+import (
+	"syscall/js"
+)
+
+func RunVMCode(jsCode string) {
+	js.Global().Call("eval", jsCode)
+}
+
+func GenerateChainCode(httpRespBody string) string {
```

Comment:
can you add a comment on what this function is doing? what is  generate chain code?